### ISSUE TITLE
A11y_ML .NET Model Builder_Image Classification-Next Steps_ScreenReader: Narrator focus is not landing on Deploy your model/Train in Azure/Improve your model cards and no information is provided by Narrator regarding those cards.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1634]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #A11yPreview; #Narrator; #Screen Reader; 

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Turn on NVDA. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Image classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In Data tab, Tab till "Browse" button and select the input file
7. After uploading data, click next step button, start training in Train tab
8. After completing training, click next step button, complete all the steps and navigate to 'Next Steps' tab
9. Verify whether narrator focus goes to cards or not

### Actual Result:
Narrator focus is not landing on Deploy your model/Train in Azure/Improve your model cards and no information is provided by Narrator regarding those cards.

### Expected Result:
Narrator focus should not land on Deploy your model/Train in Azure/Improve your model cards and Narrator should announce clearly regarding those cards when focus lands on the cards.

### User Impact:
Screen reader users will not know the content present in 'Next steps' tab if narrator is not announcing anything about the cards

**Comment by beccamc** [comment]
Validate on  rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Narrator focus is landing on all controls and also announcing about the controls in 'Next steps' tab. Hence closing the issue.

https://user-images.githubusercontent.com/83570638/131463310-9466f2f9-16e3-436a-90db-70704766af51.mp4

**Comment by akashelijah05** [comment]
#Closed;
